<launch>
	<!-- <arg name="platoon_state_topic_name" default="/drivers/v2v" /> -->
	<arg name="platoon_state_topic_name" default="/carla/platoon_state" />

    <node name="waypoint_reconstructor" pkg="waypoint_reconstructor" type="waypoint_reconstructor" output="screen"  ns="/plauto">
		<rosparam command="load" file="$(find waypoint_reconstructor)/config/waypoint_reconstructor.yaml" /> <!--Load parameters from config files-->
		<!-- <rosparam command="load" file="$(find gps)/config/gps_config.yaml"/> -->
		<param name="platoon_state_topic_name" value="$(arg platoon_state_topic_name)" />
	</node>
</launch>
